plane/source/docs/apms-failsafe-function.rst

```diff
@@ -100,21 +100,21 @@ Meaning that when flying, our throttle values will range between 1100 -
 #. Turn off the transmitter. You should see the value drop
    significantly. This will be the PWM value relayed to the autopilot in
    the event RC link was lost during flight.
-#. Make sure THR_FS_VALUE is an adequate number to trigger the
+#. Make sure :ref:`THR_FS_VALUE<THR_FS_VALUE>` is an adequate number to trigger the
    failsafe function on the autopilot.
-#. Make sure FS_SHORT_ACTN and FS_LONG_ACTN are both enabled (set to
-   1).
+#. Make sure :ref:`FS_SHORT_ACTN<FS_SHORT_ACTN>` and :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` are both enabled (set to
+   a non-zero value).
 #. Connect on the mission planner with your RC transmitter on. Verify on
    the bottom right corner of the HUD that you are “flying” in a non
    auto mode (Manual, Stabilize, FBW are ok).
-#. Turn off your transmitter. After S_SHORT_TIMEOUT sec the flight mode should
-   switch to Circle. After FS_LONG_TIMEOUT sec the flight mode should switch to RTL.
+#. Turn off your transmitter. After :ref:`FS_SHORT_TIMEOUT<FS_SHORT_TIMEOUT>` sec the flight mode should
+   switch to Circle. After :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` sec the flight mode should switch to RTL.
```

Comment:
The switch depends on what the action is, won't always be circle, unless you are committing to that for the example (in which case it should be noted)

Reply:
updated